region/brinstar/red/Red Tower.json

```
@@ -1435,52 +1432,112 @@
+        "Space jump up near the shot block, then shoot it while falling right as it goes off screen, then space jump back up.",
+        "To get up with no other items, it is best to use well-timed short jumps to go up as rapidly as possible."
+      ]
+    },
+    {
+      "link": [9, 5],
+      "name": "Jump Into IBJ",
+      "requires": [
+        "canJumpIntoIBJ",
         {"or": [
           "canBombAboveIBJ",
           {"ammo": {"type": "PowerBomb", "count": 1}}
         ]},
+        {"obstaclesCleared": ["B"]}
+      ],
+      "note": [
+        "Jump into IBJ to avoid breaking the ledges. Place a Power Bomb on the ascent to break the block and continue the IBJ through it.",
+        "It is possible to do without a Power Bomb with canBombAboveIBJ."
+      ]
+    },
+    {
+      "link": [9, 5],
+      "name": "Long IBJ With Broken Ledges",
+      "requires": [
+        "canBeVeryPatient",
+        "h_canIBJ",
+        {"ammo": {"type": "PowerBomb", "count": 1}},
+        {"or": [
+          {"obstaclesCleared": ["B"]},
+          {"ammo": {"type": "PowerBomb", "count": 2}}
+        ]}
+      ],
+      "note": [
+        "Without canJumpIntoIBJ, Samus will need to kill the Rippers while breaking the bomb blocks then start from the platforms below.",
+        "Place a Power Bomb on the ascent to break the block and continue the IBJ through it."
+      ]
+    },
+    {
+      "link": [9, 9],
+      "name": "Kill the Rippers - Power Bomb Without Breaking the Ledges",
+      "requires": [
+        "h_canUsePowerBombs",
+        "canCarefulJump",
         {"or": [
+          "canWalljump",
+          "canSpringBallJumpMidAir",
+          "SpaceJump",
           {"and": [
-            {"or": [
-              "canWalljump",
-              "canTrickyDashJump",
-              "canSpringBallJumpMidAir"
-            ]},
-            {"ammo": {"type": "PowerBomb", "count": 1}}
+            "HiJump",
+            "canTrickyDashJump"
           ]},
-          {"enemyKill": {
-            "enemies": [
-              ["Ripper", "Ripper", "Ripper"],
-              ["Ripper"]
-            ],
-            "explicitWeapons": ["Super", "PowerBomb"]
-          }},
-          {"obstaclesCleared": ["B"]}
+          {"and": [
+            "h_canJumpIntoIBJ",
+            {"or": [
+              "canStaggeredIBJ",
+              "canDoubleBombJump"
+            ]}
+          ]}
+        ]}
+      ],
+      "clearsObstacles": ["B"],
+      "note": "To kill the Rippers with a single Power Bomb without breaking the ledge, place the bomb just above the second Ripper."
+    },
+    {
+      "link": [9, 9],
+      "name": "Kill the Rippers - Supers",
+      "requires": [
+        {"ammo": {"type": "Super", "count": 4}},
+        {"or": [
+          "canWalljump",
+          "SpaceJump",
+          "h_canJumpIntoIBJ",
+          {"and": [
+            "canSpringBallJumpMidAir",
+            {"or": [
+              "HiJump",
+              "canTrickyDashJump"
```

```suggestion
              "canTrickyJump"
```
A dash jump doesn't help, but you can blindly shoot at the top ripper